resources/blueGreenCheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+# this is simply testing if the application root returns HTTP 200
+curl -so /dev/null -w '%{response_code}' https://$1 | grep 200
```

Comment:
We have a smoke test script that supports specifying the status code

Reply:
for sure we can add this instead and then pass the status code via `dockerEnvVars` to the `dockerExecute` step